UPGRADE_GUIDE.md

```diff
@@ -2,6 +2,13 @@
 
 ## From `0.3.9` to `0.3.10`
 
+### AuthorityLossImminent support removed
+
+The GDK for Unity no longer supports the AuthorityLossImminent authority state. If this feature is currently enabled, it will not trigger the authority state for any reader or writer. The methods to `Ack` the authority state has also been removed.
+
+If you currently make use of this feature, we recommend using the `MonoBehaviour.OnDisabled` method instead.
```

Comment:
This probably isn't great advice for some use cases.

For example, if you have something which tries to sync some local state when loss is imminent, by the time this is called the authority is already gone.

Reply:
Any suggestions? Because I can't really think of any

Reply:
As discussed offline, removing the "advice"